signale.js

```diff
@@ -161,12 +162,12 @@ class Signale {
       if (args[0] instanceof Error) {
         [msg] = args;
       } else {
-        const [{prefix, message, suffix}] = args;
```

Comment:
Looks nice! The only thing left is to bring back `message` for destructuring `args` :

```js
const [{prefix, message, suffix}] = args;
```

and directly apply interpolation on it on:

```js
msg = util.format(...message);
```

Thank you a lot for taking the time to help out : )

Reply:
I am getting `{prefix, message, suffix}` as undefined all the times

Reply:
Yep, it is only applicable when using the `prefix` and `suffix` attributes, for example:

```js
const signale = require('signale');

signale.complete({prefix: '[task]', message: ['Fix issue #%s', 59], suffix: '(@santa)'});
```

Reply:
now I have to check undefined case.

Reply:
Done. `message` for destructuring `args`